packages/nuxt/src/runtime/plugins/sentry.server.ts

```diff
@@ -29,10 +29,16 @@ export default defineNitroPlugin(nitroApp => {
       captureContext: { contexts: { nuxt: structuredContext } },
       mechanism: { handled: false },
     });
+
+    vercelWaitUntilAndFlush();
```

Comment:
should we have to await `waitUntil` or am I misunderstanding how this should work?

Reply:
waitUntil is a worker API that simply tells the runtime not to freeze execution until the passed promise has resolved.